documentation/misc/Update_Dependencies_Lotus.md

```diff
@@ -22,6 +22,10 @@ Updating these dependencies in Lotus is usually related to network upgrades.  Se
 
 ## Updating Filecoin-FFI
 
+0. In Lotus´s [go.mod file](https://github.com/filecoin-project/lotus/blob/master/go.mod), search for `filecoin-ffi` and update the version to your wanted version.
```

Comment:
@rjan90 : I added this for discussion.  It wasn't clear to me how the ffi version in Lotus go.mod comes into play, but I assume it's good to keep these in sync?

Reply:
I think the filecoin-ffi version in Lotus go.mod might just be some relic/historic thing, and probably redundant given that we are using replace a bit higher: https://github.com/filecoin-project/lotus/blob/d017c15cb34edfe7757c8089afbc915f8255ce76/go.mod#L13-L14.

We can keep these in sync for now, and see if can just remove in the go.mod require section. Added a comment here: